release/k3s/k3s.go

```diff
@@ -230,6 +231,8 @@ func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]st
 	return tags, nil
 }
 
+// The getAuth privateKey is supplied by user in the Release struct variable ssh_key_path.
```

Comment:
Please provide a doc comment for the function itself while mentioning how it handles it's argument.

This needs to be in doc comment format.

```go
/// nameOf Func ...
func nameOfFunc() {}
```

Reply:
This is complete.